### PR TITLE
Add translation for Ruby Prize 2015 (en)

### PR DESCRIPTION
**Pull request by zzak**
/cc @snoozer05 @hsbt

### CODE REVIEW

File: en/news/_posts/2015-06-30-ruby-prize-2015.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+layout: news_post
+title: "Ruby Prize 2015 推薦受付開始"
```
Comment (amatsuda) [inline en/news/_posts/2015-06-30-ruby-prize-2015.md]: Shouldn't this be translated?

Comment (zzak) [inline en/news/_posts/2015-06-30-ruby-prize-2015.md]: Haha, whoops! Good catch @amatsuda!

**Comment by snoozer05** [comment]
thx :star2:

**Comment by hsbt** [comment]
@zzak Thanks!

File: en/news/_posts/2015-06-30-ruby-prize-2015.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+title: "Ruby Prize 2015 nominations are now open"
+author: "Koji Shimada"
+translator: "zzak"
+date: 2015-06-30 19:15:00 +0000
+lang: en
+---
+
+Ruby Prize 2015 nominations are now being accepted for new and outstanding members in the Ruby community.
+
+The Ruby Prize is given to recognize the efforts of remarkable activity and achievements in the Ruby Community. The prize will be decided by the executive committee comprised of three parties, the Ruby Association, Nihon Ruby no Kai and Matsue City.
```
Comment (BanzaiMan) [inline en/news/_posts/2015-06-30-ruby-prize-2015.md]: How about: "decided" → "awarded"

File: en/news/_posts/2015-06-30-ruby-prize-2015.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+Ruby Prize winner and final nominee (1-2 people) will receive an award at the RubyWorld Conference 2015, to be held in Matsue, Shimane Prefecture on November 12th & 13th.
```
Comment (BanzaiMan) [inline en/news/_posts/2015-06-30-ruby-prize-2015.md]: The original text has "招待して", which seems to imply that these nominees will be invited to the conference. (It is not clear, though, if the travel expenses are covered.)

File: en/news/_posts/2015-06-30-ruby-prize-2015.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+In addition, the Ruby Prize winner will be awarded sub-prize money of 1 million yen (approx. 8,213 USD at exchange rates as of June 30th, 2015).
```
Comment (BanzaiMan) [inline en/news/_posts/2015-06-30-ruby-prize-2015.md]: "be awarded sub-prize money of" → "also be awarded"

**Comment by zzak** [comment]
@BanzaiMan Thanks for your review!

**Pull request merged**